circle.yml

```diff
@@ -1,6 +1,11 @@
 machine:
   node:
     version: 6.2.1
+
+dependencies:
+  post:
+    - npm install react react-native @accounts/common @accounts/client
```

Comment:
instead of doing that you maybe should add devDependencies as the same as peerDependencies